mail/2.0/_index.md

```diff
@@ -33,19 +33,21 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
-
-| Representative                                 | Representative for: | Vote |
-|------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
+The Specification Committee Ballot concluded successfully on 2020-09-11 with the following results.
+
+| Representative                                      | Representative for: | Vote |
+|-----------------------------------------------------|---------------------|------|
+| Kenji Kazumura                                      | Fujitsu             |  +1  |
+| Dan Bandera, Kevin Sutter                           | IBM                 |  +1  |
+| Ed Bratt, Dmitry Kornilov                           | Oracle              |  +1  |
+| Andrew Pielage, Matt Gill                           | Payara              |  +1  |
+| Scott Stark, Mark Little                            | Red Hat             |  +1  |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |  +1  |
```

Comment:
I was going to let this one go, but since I found the missing url below... The only two members of Tomitribe that can vote are David and Jean-Louis.  Cesar is extremely helpful with reviewing and prepping, but the votes are only coming from David and Jean-Louis.  Remove Cesar from this list.  Thanks.

Reply:
Ok, I took Authorizaiton as a template because that closed just 2 days earlier, but maybe it's not necessary to reopen that, otherwise they might have to change it as well.

Reply:
Please have a look, somehow Egit is totally crazy and won't recognize real changes, but I also removed Cesar and had to remove a trailing line, but those should be ignored by rendering I hope. Mailing list link also there, I did not touch the date in the first section because no spec I saw did change it based on the ballot or last deploy date, e.g. Bean Validation has "date: 2020-07-07" and "The Specification Committee Ballot concluded successfully on 2020-08-26", so the July date wasn't changed so far.

Reply:
Fixed now